src/styles/layout.scss

```diff
@@ -10,6 +10,7 @@ body {
   font-family: var(--base-type-face);
   margin: 0;
   margin-top: var(--nav-height);
+  overflow-x: hidden;
```

Comment:
Setting `overflow-x:hidden` does solve the issue, but I consider it a dirty solution. A better solution would be to check which element is causing the horizontal scroll and fix it.